pom.xml

```diff
@@ -44,6 +44,31 @@
             <artifactId>fop</artifactId>
             <version>2.9</version>
         </dependency>
+        <dependency>
+            <groupId>org.apache.maven.plugins</groupId>
+            <artifactId>maven-compiler-plugin</artifactId>
+            <version>3.13.0</version>
+        </dependency>
+
+
+        <dependency>
+            <groupId>org.apache.tika</groupId>
+            <artifactId>tika-core</artifactId>
+            <version>2.9.2</version>
+        </dependency>
+        <dependency>
+            <groupId>org.apache.tika</groupId>
+            <artifactId>tika-langdetect</artifactId>
+            <version>2.9.2</version>
+            <type>pom</type>
+        </dependency>
+        <dependency>
+            <groupId>org.apache.tika</groupId>
+            <artifactId>tika-langdetect-optimaize</artifactId>
+            <version>2.9.2</version>
+        </dependency>

```

Comment:
Have you looked at issue #3 - is fop still not included? Are the tika classes? (If so - then we can close that issue when this merges).

Reply:
fop is included. I did not have problems with it.

Reply:
Got it - guessing it's a build command problem - I made a note in the issue #3.